packages/cli/test/integration/public-api/projects.test.ts

```diff
@@ -149,14 +144,13 @@ describe('Projects in Public API', () => {
 			/**
 			 * Arrange
 			 */
-			const owner = await createOwner({ withApiKey: false });
 			const projectPayload = { name: 'some-project' };
 
 			/**
 			 * Act
 			 */
 			const response = await testServer
-				.publicApiAgentFor(owner)
+				.publicApiAgentWithApiKey('')
```

Comment:
Maybe we could have `publicApiAgentWithoutApiKey()`? Would make this a bit cleaner and having the header with empty value is after a different that having no header at all